### ISSUE TITLE
Counties where after fixing typos, county totals won't match official county totals

### ISSUE THREAD

**Issue by peter-redistricting-data-hub** [openelections/openelections-data-pa#144]
Hopefully this will prevent someone from doing extra work:

Here are the counties that I have confirmed will have differences:
- Berks (011)
- Crawford (039)
- Juniata (067)
- Mifflin (087)
- Montour (093)
- Somerset (111)

County that I am almost positive will have differences
- Blair (013)

The differences in totals are relatively minor (<15 votes in a race) in every county except for Mifflin, where the largest difference is 494 votes for the republican candidate in state legislative seat 171. As the Mifflin report seems to be official, I am not sure why there are differences.

If someone is interested in results that match the official totals (or would like to add those in for their file, I would point them to this file: https://github.com/openelections/openelections-sources-pa/blob/master/2018/ERStat_2018_G(114234)_20190726.txt).

I would caution any user of that file that the quality seems somewhat questionable IMO, compared to these reports. For example, to get the Crawford green party senate candidate totals to match up, two precincts lose 6 and 4 votes for this candidate. To me, this seemed somewhat unlikely as all other precincts contained votes for the green party candidate and that is a fairly large precinct.

**Comment by peter-redistricting-data-hub** [comment]
@rootsmusic

**Comment by peter-redistricting-data-hub** [comment]
The official totals I'm referring to are available here: https://www.electionreturns.pa.gov/ReportCenter/Reports

**Comment by rootsmusic** [comment]
@peter-redistricting-data-hub  My validation is limited to Governor, U.S. Senate and U.S. House.  For these offices, I found errors in the following counties:
Crawford _fixed_ in #127.
Juniata opened in #141.
Montour opened in #142.

**Comment by peter-redistricting-data-hub** [comment]
Thanks, I'll check out what you had for those 3. They aren't necessarily *errors* in the files though. 

I think "error" implies a typo in the transcription of the reports, but you've just identified results with slight differences. I'd be curious what @dwillis has to say, but I could see an argument for not updating the correctly transcribed reports, even if they don't match the official results in another file

**Comment by rootsmusic** [comment]
@peter-redistricting-data-hub Everyday during canvassing, counties update their election results daily.  At the end of the canvassing period, each county certifies its final version of election results.  That certified version is then certified again by the Secretary of State.  (If a county discovers an error after it certified, then the county can notify the Secretary to make a correction before the state's certification.)

So the only two versions of a county's election results that are certified are the county's own certification and the state's certification.  No other version is certified and other versions are void as soon as one version is certified.  The county certifies that canvassing was properly and legally completed, and the state certifies that there are no election irregularities (e.g. [NC-09](https://ballotpedia.org/North_Carolina's_9th_Congressional_District_election,_2018) in 2018 General Election).

Your issue #136 found the "Final Results", as labeled and dated on 11/9/2018.  [OE's preliminary version](https://github.com/openelections/openelections-sources-pa/blob/master/2018/general/Forest_PA_2018_general.html) (before your issue was fixed) had been dated on election day.  If OE wants to also transcribe the preliminary version, then that version's data should be clearly labelled as "uncertified".

**Comment by dwillis** [comment]
Thanks to both of you for your assistance in identifying errors and for this discussion. Our goal is to have the certified results converted into data, and for Forest we didn't check again later to verify that a better version had been posted, so @rootsmusic is correct on that score. In my experience, the DOS-provided statewide files (the `ERStat` ones) do not exactly match official county results in every case, which is one reason we started collecting from each county in the first place (the other being better data on votes by mode).

**Comment by rootsmusic** [comment]
@peter-redistricting-data-hub  Is this still an issue?  VEST could have been clearer, but [their documentation](https://redistrictingdatahub.org/wp-content/uploads/2020/12/readme_pa_vest_18.txt) acknowledged: "The candidate totals for Berks, Blair, Crawford, and Mifflin differ from the county totals reported by the state and therefore the statewide totals differ from the official results accordingly."  This acknowledgement referred to "the county totals reported by the state" as "[the statewide totals](https://electionreturns.pa.gov/ReportCenter/Reports)".  So "the official results" is referring to those officially reported by the county to the State's DOS, which are the results certified by every county in this repo.

**Comment by peter-redistricting-data-hub** [comment]
This wasn't an issue with the OpenElections file per say, just hopefully a warning to save someone time if they took it open themselves to sum the transcribed totals here and compare them against those of the state.

As an example, in Berks county, even though the precinct results are "certified precinct results" the precinct-level totals are different from the state's "certified" county-level totals for Tom Wolf

Does that make sense?

<img width="1118" alt="Screen Shot 2022-10-04 at 7 46 43 PM" src="https://user-images.githubusercontent.com/72451074/193950882-d78c0367-263e-4b82-9b41-41ff34ff7e27.png">
<img width="1013" alt="Screen Shot 2022-10-04 at 7 46 15 PM" src="https://user-images.githubusercontent.com/72451074/193950884-ab8363b9-d2c0-4546-ad2e-f3b23a7792ac.png">

**Comment by dwillis** [comment]
@peter-redistricting-data-hub right, exactly. we undertook converting county-generated files _knowing_ that they sometimes would not match state totals.
